frontend/src/containers/TeaserPage/TeaserStudent.tsx

```diff
@@ -16,6 +17,7 @@ import {
 } from '../../data/StudentContent'
 import { initializeParams } from '../../redux/reducers/SearchParamsReducer'
 import { useAppDispatch } from '../../redux/store'
+import { queryAllByAltText, queryByRole } from '@testing-library/react'
```

Comment:
Pakidelete nalang ito since hindi naman siya ginagamit. Pero if ginagamit, tuloy lang.

Also, if may variables and imports na hindi ginagamit, pakidelete nalang din.